hole-punch-interop/helpers/cache.ts

```diff
@@ -73,6 +73,11 @@ async function loadCacheOrBuild(dir: string, ig: Ignore) {
     const cacheKey = await hashFiles(files)
     console.log("Cache key:", cacheKey)
 
+    if (AWS_BUCKET === "") {
```

Comment:
This will catch `undefined`, `null` and `''`.  As written it'll only catch `''`.

```suggestion
    if (!AWS_BUCKET) {
```